product_supplierinfo_discount/models/__init__.py

```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
 # © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
 #        Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 from . import product_supplierinfo
 from . import purchase_order
-from . import procurement_order
+from . import procurement
```

Comment:
why don't you rename to procurement_rule file ?

Reply:
I renamed it to procurement because in the odoo's code base it is named procurement.

Reply:
yes. Well in OCA modules, rules are more precise. You can check them here : 
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming

Reply:
No problem, I will rename it :)